test/go/table_schema_cache_test.go

```diff
@@ -107,8 +107,8 @@ func (this *TableSchemaCacheTestSuite) TestLoadTablesRejectTablesWithoutNumericP
 	tableSchemaCache, err := ghostferry.LoadTables(this.Ferry.SourceDB, this.tableFilter, nil, nil, nil, nil)
 
 	this.Require().Nil(err)
-	this.Require().Contains(tableSchemaCache, testhelpers.TestSchemaName + "." + table)
-	this.Require().Equal(1, len(tableSchemaCache[testhelpers.TestSchemaName + "." + table].PKColumns))
+	this.Require().Contains(tableSchemaCache, testhelpers.TestSchemaName+"."+table)
+	this.Require().Equal(1, len(tableSchemaCache[testhelpers.TestSchemaName+"."+table].PKColumns))
```

Comment:
Whitespace change on purpose?

Reply:
I ran the default Go formatter on all these files. I also dislike how they collapse spaces around these operators, but Go formatter doesn't allow you to customize because all Go code must look the same i guess.